packages/next/client/index.tsx

```diff
@@ -612,7 +617,27 @@ async function doRender({
       }
       return promise
     })
-    return Promise.all(required)
+    return Promise.all(required).catch(() => {
```

Comment:
IIRC NextJS polyfills `allSettled`. Would that be a better alternative?

Reply:
Not in this case, we only care if one of these errors and immediately proceed to reload the page.

AFAIK, `Promise.all` will bail early on the first error, which is what we want here.